pyscriptjs/tests/integration/support.py

```diff
@@ -177,6 +177,10 @@ def wait_for_pyscript(self, *, timeout=None, check_errors=True):
             timeout=timeout,
             check_errors=check_errors,
         )
+        # We need this very small wait time for pyscript to append
+        # the script to the page, otherwise clicks and other actions
+        # won't be registered in the test.
+        self.page.wait_for_timeout(100)
```

Comment:
I think this comment is wrong, see above for my thoughts about it.
But I don't want to block this PR because of this: I think that for now it's enough to write a comment which says that we don't know why it's needed.

Reply:
Thank you again, I'll change the comment saying that we don't know why we need this 😄